src/ccmain/tesseractclass.cpp

```diff
@@ -477,7 +481,10 @@ Tesseract::~Tesseract() {
   for (auto *lang : sub_langs_) {
     delete lang;
   }
-  delete lstm_recognizer_;
+  for (int i = 0; i < lstm_recognizers_.size(); ++i) {
```

Comment:
`i` should use the same data type as the return value of `size()`.

Reply:
`for (auto &&r : lstm_recognizers_) delete r;`

Reply:
Thanks for your review comments. I have addressed this now.